src/main/java/seedu/mark/logic/commands/AnnotationCommand.java

```diff
@@ -76,3 +95,17 @@ public ParagraphIdentifier getPid() {
     }
 
 }
+
+/**
+ * This is a dummy pid for a non-existent paragraph, null.
+ */
+class DummyParagraphIdentifier extends ParagraphIdentifier {
+    public DummyParagraphIdentifier() {
+        super(Index.fromOneBased(1), ParagraphType.STRAY);
+    }
+
+    @Override
+    public String toString() {
+        return "NULL (i.e. your note has been added to the general ";
```

Comment:
Is this missing a closing bracket? (I can't find where it's used ^^")

Reply:
Oops typo. It's used when the users add to paragraph `p/null`; added the bracket